go.mod

```diff
@@ -16,12 +16,20 @@ require (
 	github.com/mattn/go-colorable v0.1.2 // indirect
 	github.com/mattn/go-runewidth v0.0.4 // indirect
 	github.com/olekukonko/tablewriter v0.0.1
+	github.com/onsi/ginkgo v1.11.0
+	github.com/onsi/gomega v1.7.0
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
 	github.com/testcontainers/testcontainers-go v0.0.10
-	golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456 // indirect
-	golang.org/x/tools v0.0.0-20200116203608-1c4842a210a7 // indirect
-	gopkg.in/yaml.v2 v2.2.4
+	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550 // indirect
+	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
+	golang.org/x/sync v0.0.0-20190423024810-112230192c58 // indirect
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.18.4 // indirect
```

Comment:
Can we avoid taking a dependency on the k8s libraries?

Reply:
Surer. I remove most of the k8s libraries. But the client_credenctials_flow uses the "k8s.io/utils/clock".